backend/isoko-backend/src/handlers/triggers/cognito-post-confirmation.js

```diff
@@ -23,7 +23,24 @@ exports.cognitoPostConfirmationHandler = async (event, context, callback) => {
       ReturnConsumedCapacity: 'TOTAL',
       TableName: USER_TABLE,
    };
-   await docClient.put(params).promise();
-
-   callback(null, event);
+   
+   let userResult; 
+   try {
+      const dynamoResult = await docClient
+         .put(params)
+         .promise();
+      userResult = dynamoResult.Items; 
+   } catch(e) {
+      console.log(`PUT error: ${e}`); 
+   }
+   
+   const response = {
+      statusCode: 200, 
+      body: {
+         results: userResult,
+      },
+   }
+   
+   // callback(null, event);
+   return response; 
```

Comment:
I'm a little confused about this change, was this supposed to be included in this pull request?

Reply:
i am also confused ... i just got this error when i first ran 'npm test' after branching and wasn't sure what caused it bc i didn't change anything in that file

<img width="1479" alt="Screen Shot 2022-01-30 at 7 07 54 PM" src="https://user-images.githubusercontent.com/43476619/151920722-9290f757-54fb-4a89-96bd-bf62c53940a7.png">
 
adding a try-catch made the error go away but i don't think the resulting behavior is what we want bc it just caught the same error: 

`TypeError: Cannot read properties of undefined (reading 'promise')`

and the test case wasn't checking for an error being thrown. should def talk abt it more in class tomorrow